types/ember__helper/package.json

```diff
@@ -8,8 +8,8 @@
         "https://github.com/emberjs/ember.js"
     ],
     "dependencies": {
-        "@glimmer/manager": ">= 0.83.0",
-        "@glimmer/runtime": ">= 0.83.0",
+        "@glimmer/manager": ">=0.83.0 <0.93.0",
```

Comment:
oh! we don't need `||` ? neat

Reply:
This is an `>=0.83.0 %% <0.93.0`, it's just that `&&` is implict.